AGENTS.md

```
@@ -0,0 +1,7 @@
+<!-- convex-ai-start -->
+This project uses [Convex](https://convex.dev) as its backend.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)